frontend/src/styles/News/News.scss

```diff
@@ -1,7 +1,9 @@
 .news-title{
     color: white;
-    font-size: 35px;
-    margin: 5vh 0vw 5vh 1vw;
+    font-size: 40px;
```

Comment:
We need to change the px to either vw/vh/% to keep it consistent with the rest of the files.

Reply:
Also I'm pretty sure I made the change to "margin: 5vh 0vw 5vh 1vw" instead of padding in my PR @JScheire so why did we change it back?

Reply:
Whoops I think this is a mistake